static/src/javascripts/projects/common/modules/commercial/dfp/dfp-api.js

```diff
@@ -783,7 +781,7 @@ define([
             rendered = false;
             adverts = {};
             slotsToRefresh = [];
-            mediator.off('window:resize', windowResize);
+            mediator.removeAllListeners('window:resize');
```

Comment:
Isn't there a risk of removing the listeners set elsewhere? why not keep the `windowResize` but assign it inside `refreshOnResize`?